overlay/make-package-set/full.nix

```diff
@@ -11,6 +11,7 @@
   packageFun,
   cargo,
   rustc,
+  rust-src,
```

Comment:
Let's just get rid of these and take `rustChannel` for simplicity.

Reply:
Just to simplify arguments?